src/mlpack/tests/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ add_executable(mlpack_test
   sparse_coding_test.cpp
   spill_tree_test.cpp
   split_data_test.cpp
+  string_utility_test.cpp
```

Comment:
Probably it's better to dedicate this test to the dictionary encoding class rather than the project itself. I mean something like `dictionary_encoding_test.cpp`.

Reply:
You mean, for each class or functionality build a new one, dictionary encoding, bag of word encoding, string cleaning, tfidf and so ?

Reply:
Yes, I suggest to write the tests for each class in a separate file.